cpp/ql/test/library-tests/valuenumbering/GlobalValueNumbering/ir_gvn.expected

```diff
@@ -323,21 +323,21 @@ test.cpp:
 #   45|     r45_6(glval<int>)           = VariableAddress[global03] : 
 #   45|         valnum = r43_6
 #   45|     r45_7(int)                  = Load                      : &:r45_6, ~m39_2
-#   45|         valnum = unique
+#   45|         valnum = r43_7
```

Comment:
The library now seems to claim that the loads from `global03` on lines 43 and 45 give the same value, but that's not true. I suppose that's due to #2696, so it's not the fault of this PR, but it's worth mentioning as a comment in the test.

Reply:
I've added a comment in `test.cpp` mentioning #2696 now.